main.go

```diff
@@ -197,6 +198,8 @@ func main() {
 
 	runner := NewDumpRunner(basePath)
 
-	log.Print("Running tasks...")
-	RunAllTasks(runner, basePath, *concurrentTasks)
+	log.Print("Running dump tasks...")
```

Comment:
Do we really want to bother users about "dump" x "analysis" tasks? Those seem like internal implementation details to me.

The idea of printing "Doing XYZ" and then a "." every now and then was just as a poor man's progress bar.